src/psyclone/tests/psyir/transformations/kernel_transformation_test.py

```diff
@@ -35,6 +35,7 @@
 # Modified by: R. W. Ford, STFC Daresbury Lab
 #              I. Kavcic, Met Office
 #              S. Siso, STFC Daresbury Lab
+#              J. Henrichs, Bureau of Meteorology
```

Comment:
Is this a correction to a previous oversight? If so, please could you rm L42 please.

Reply:
Technically I had made some modifications before, but I don't think it was justified to put my name in. Now I have done some code additions, so I added my name. Hope that's ok. And yes, I removed L42 :)